### DISCUSSION TITLE
Additional maintainers

### DISCUSSION THREAD

**Discussion by julien-duponchelle** [julien-duponchelle/python-mysql-replication#487] [General]
Hello,

I feel that I'm currently a bottleneck on the contribution pipeline of the project. Who would be interested in the contributors to get merge rights ?

Best,

Julien

**Comment by dongwook-chan** [comment]
I recommend @sean-k1 for the maintainer role. He has conducted a comprehensive walkthrough of the code and understands every aspect of it. His involvement would significantly benefit the project.

**Reply by julien-duponchelle** [reply]
@sean-k1 are you happy with that?

**Reply by sean-k1** [reply]
@julien-duponchelle sure!

**Reply by julien-duponchelle** [reply]
Invitation sent

**Reply by sean-k1** [reply]
@julien-duponchelle 
Sorry Julien, but I have an error in pypi version 0.45.0, 1.0.0 ,1.0.1 and I don't have authorization to delete it, could you please delete it?

**Reply by julien-duponchelle** [reply]
Should be good now

**Comment by dongwook-chan** [comment]
I recommend @dongwook-chan too.

**Reply by julien-duponchelle** [reply]
@dongwook-chan are you happy with that?

**Reply by dongwook-chan** [reply]
Yes!

**Reply by julien-duponchelle** [reply]
Invitation sent!

**Reply by dongwook-chan** [reply]
@julien-duponchelle 
Thank you for granting me collaborator rights on the python-mysql-replication project. 😁 I truly appreciate the trust and opportunity you've extended. However, I noticed that I was given "collaborator" status rather than the "maintainer" role as previously discussed. I wanted to check if this was intentional or an oversight. Having maintainer rights would assist in streamlining some of the tasks I anticipate, especially with the upcoming Hacktoberfest preparations.

**Reply by julien-duponchelle** [reply]
I'm happy with that but I don't know how to do it. I see only the option to add collaborator but I don't see a way to increase the permission. I wonder If I need to move the project to an org for that.

**Comment by jaehyeonpy** [comment]
I think providing project philosophy/goal and principles for maintainers to follow can help for recruiting additional maintainers!

**Reply by julien-duponchelle** [reply]
That's a good idea. Maybe a code of conduct could help. Do you have an example ?  I think I could draft something and share it the community.

**Reply by jaehyeonpy** [reply]
I think your suggestion is really a good one, but I'd like to propose this before we go into details:

If you want to solve problems like async ones as you said, then you can refer to examples about management other open source project maintainers did. Their trial and error would be helpful. 

However, if you want to expand the project and have it become a part of an industry or other, you can refer to documents related to community organization. Approaches such as community organization would be more beneficial in this case than relying on the personal experiences of other project maintainers.

You can decide on the direction you want to take on your own, or with the new additional maintainers @dongwook-chan @sean-k1. I can find and share some example or document, if needed, after the decision on the direction has been made.

**Comment by dongwook-chan** [comment]
@julien-duponchelle 
I completely understand the importance of deliberation in making such decisions and I want you to take all the time you need. However, I want to apply python-mysql-replication for Hacktoberfest and there are certain modifications that need to be implemented beforehand. To ensure we're in good standing for Hacktoberfest, I'd like to request maintaining rights, even if on a temporary basis, so we can make the necessary changes. The application ends this month.
